### PR TITLE
Update Playwright config and workflow

### PR DESCRIPTION
**Pull request by ProchaLu**
This PR updates the Playwright configuration and workflow file to be more efficient and effective.

### CODE REVIEW

File: .github/workflows/test-playwright-and-deploy-to-fly-io.yml [hunk 1]
```diff
@@ -2,10 +2,9 @@ name: Playwright Tests and Deploy to Fly.io
 on: push
 
 jobs:
-  e2e-tests:
-    name: Playwright test
+  playwright-tests:
+    name: Playwright Tests
     runs-on: ubuntu-latest # or macos-latest, windows-latest
-    # Update env variables with your own database credentials
```
Comment (karlhorky) [inline .github/workflows/test-playwright-and-deploy-to-fly-io.yml:8]: why was this removed?

File: .github/workflows/test-playwright-and-deploy-to-fly-io.yml [hunk 2]
```diff
@@ -26,19 +25,20 @@ jobs:
...
         with:
           node-version: 'lts/*'
           cache: 'pnpm'
+
       - name: Install dependencies
         run: pnpm install
       - name: Install SafeQL if not yet installed (eg. on Windows dev machines)
         run: grep package.json -e '@ts-safeql/eslint-plugin' || pnpm add @ts-safeql/eslint-plugin libpg-query
       - run: pnpm migrate up
-      # Also generates next-env.d.ts, required for tsc
```
Comment (karlhorky) [inline .github/workflows/test-playwright-and-deploy-to-fly-io.yml]: why was this removed?

**Review by karlhorky — APPROVED** [review]

File: .github/workflows/test-playwright-and-deploy-to-fly-io.yml (hunk 1 above)
Comment (ProchaLu) [reply .github/workflows/test-playwright-and-deploy-to-fly-io.yml:8]: Reverted the deleted comment
[2309a75](https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/pull/15/commits/2309a75ca762b3d5bf0ecbc7cfecf18baa5f05d8)

File: .github/workflows/test-playwright-and-deploy-to-fly-io.yml (hunk 2 above)
Comment (ProchaLu) [reply .github/workflows/test-playwright-and-deploy-to-fly-io.yml]: Reverted the deleted comment
[ebf3118](https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/pull/15/commits/ebf3118340c41006824dcad6dd0d576059a1e4a0)
